master/buildbot/master.py

```diff
@@ -339,6 +347,19 @@ def reconfigService(self, new_config):
             self.db_loop = task.LoopingCall(self.pollDatabase)
             self.db_loop.start(self.configured_poll_interval, now=False)
 
+    def changeServicesStateFinished(self):
+        self.change_services_state = False
+        self.change_service_lock.release()
+
+    @defer.inlineCallbacks
+    def changeServicesStateStarted(self):
+        if not self.change_service_lock.waiting:
+            yield self.change_service_lock.acquire()
+            self.change_services_state = True
+
+    def changeServicesStateRunning(self):
+        return self.change_services_state
```

Comment:
Do we really need a getter method for this?

Why not just access `master.change_services_state directly`?